### DISCUSSION TITLE
add string from tele/.../SENSOR in variable

### DISCUSSION THREAD

**Discussion by DC-555** [arendst/Tasmota#19041] [Q&A] [answered]
I'm trying to add information that comes in string to a variable of type STR:

08:29:05.270 MQT: tele/Connect_V2_F870E4/SENSOR = {"Time":"2023-07-05T08:29:05","RDM6300":{"UID":"**00A7B920**"}}

the snippet I want to monitor is: UID

I will summarize my script to understand:

>D 128
RF="xxx"
>B
...
>E
...
>T
RF=RDM6300#UID
>S
print test= %RF%

user_config_override.h
#define SCRIPT_MAXSSIZE 156
#define USE_SCRIPT_FATFS_EXT
#define USE_SCRIPT_SERIAL

**Comment by gemu2015** [comment, marked as answer]
this is processed as event (when card is presented), so use section >E

**Reply by DC-555** [reply]
It works perfectly. Thank you very much.
08:30:17.455 xx
08:30:18.461 xx
08:30:19.432 xx
08:30:20.436 xx
08:30:21.441 xx
08:30:21.000 MQT: tele/Connect_V2_F870E4/SENSOR = {"Time":"2023-07-06T08:30:21","RDM6300":{"UID":"00A7B920"}}
08:30:22.445 00A7B920
08:30:23.452 00A7B920
08:30:24.459 00A7B920
08:30:25.467 00A7B920
08:30:26.474 00A7B920
